app/pages/system/UtilizationPage.tsx

```diff
@@ -215,32 +243,103 @@ function UsageTab({ silos }: { silos: SiloResultsPage }) {
         {mergedResults.map((result) => (
           <Table.Row key={result.siloName}>
-            <Table.Cell width="25%">{result.siloName}</Table.Cell>
-            <Table.Cell width="25%">{result.metrics.cpus_provisioned}</Table.Cell>
-            <Table.Cell width="25%">
-              {bytesToTiB(result.metrics.virtual_disk_space_provisioned)}
-              <span className="ml-1 inline-block text-quaternary">TiB</span>
+            <Table.Cell width="16%">{result.siloName}</Table.Cell>
+            <Table.Cell width="14%">
+              {/* dummy data for now */}
+              <UsageCell
+                numerator={result.metrics.cpus_provisioned}
+                denominator={result.metrics.cpus_provisioned}
+              />
+            </Table.Cell>
+            <Table.Cell width="14%">
+              <UsageCell
+                numerator={bytesToTiB(result.metrics.virtual_disk_space_provisioned)}
+                denominator={bytesToTiB(result.metrics.virtual_disk_space_provisioned)}
+                unit="TiB"
+              />
+            </Table.Cell>
+            <Table.Cell width="14%">
+              <UsageCell
+                numerator={bytesToTiB(result.metrics.ram_provisioned)}
+                denominator={bytesToTiB(result.metrics.ram_provisioned)}
+                unit="GiB"
+              />
             </Table.Cell>
-            <Table.Cell width="25%">
-              {bytesToGiB(result.metrics.ram_provisioned)}
-              <span className="ml-1 inline-block text-quaternary">GiB</span>
+            <Table.Cell width="14%" className="relative">
+              <AvailableCell used={8} total={60} />
+            </Table.Cell>
+            <Table.Cell width="14%" className="relative">
+              <AvailableCell used={100} total={150} unit="TiB" />
+            </Table.Cell>
+            <Table.Cell width="14%" className="relative">
+              <AvailableCell used={500} total={600} unit="GiB" />
             </Table.Cell>
           </Table.Row>
         ))}
       </Table.Body>
     </Table>
   )
 }
+
+const UsageCell = ({
+  numerator,
+  denominator,
+  unit,
+}: {
+  numerator: number
+  denominator: number
+  unit?: string
+}) => (
+  <div className="flex flex-col text-tertiary">
+    <div>
+      <span className="text-default">{numerator}</span> /
+    </div>
+    <div className="text-tertiary">
+      {denominator} {unit && <span className="text-quaternary">{unit}</span>}
+    </div>
+  </div>
+)
+
+const AvailableCell = ({
+  used,
+  total,
+  unit,
+}: {
+  used: number
+  total: number
+  unit?: string
+}) => {
+  const usagePercent = (used / total) * 100
+  return (
+    <div className="flex w-full items-center justify-between">
+      <div>
+        {used} {unit && <span className="text-tertiary">{unit}</span>}
+      </div>
+      {/* We only show the ResourceMeter if the percent crosses the warning threshold (66%) */}
+      {usagePercent > 66 && (
+        <div className="absolute right-3">
```

Comment:
Unsure about the least 'dangerous' way to do this.
We have extra padding on the trailing side of the cell but the resource meter needs to be aligned to this side and ignore that padding.

This is the simple solution, downside being that there's a risk the meter overlaps with the content of the cell.

Reply:
I think either the `absolute right-3` approach could work, or keeping it with its current setup (parent div has `justify-between` to space it out) and adding something like `transform: translateX(20px);` to shift it to the right.

To your point about the meter overlapping with the content of the cell, wouldn't the padding on the cell's trailing side prevent that collision from happening? My in-browser experiments seem to work equally well with either of the two above approaches.